docs/reference/commandline/service_update.md

```diff
@@ -21,86 +21,87 @@ Usage:  docker service update [OPTIONS] SERVICE
 Update a service
 
 Options:
-      --args command                       Service command args
-      --config-add config                  Add or update a config file on a service
-      --config-rm list                     Remove a configuration file
-      --constraint-add list                Add or update a placement constraint
-      --constraint-rm list                 Remove a constraint
-      --container-label-add list           Add or update a container label
-      --container-label-rm list            Remove a container label by its key
-      --credential-spec credential-spec    Credential spec for managed service account (Windows only)
+	  --args command                       Service command args
```

Comment:
Looks like this whole file changed from using spaces to tabs; can you revert that change?

Reply:
:scream: